### ISSUE TITLE
Guest Checkout Confusion

### ISSUE THREAD

**Issue by judygunderson** [lat9/one_page_checkout#238]
Suggest to make a separate update button which is bigger than Zen's update button - with some verbiage to click it to continue so there is no confusion on getting the rest of the form to show up and be editable.

Thanks for all your help.  Customer loved my design/colors of the css buttons as you suggested to switch to!

**Comment by lindasdd** [comment]
This was something I noticed on my first try of this plugin as well.

The easiest way to make it more clear would be to choose a much darker color for .opc-overlay.active  and to changes the button text from 'Save Changes' to something clearer such as 'Save Address and Continue Checkout' or 'Enter Address and Continue to Next Checkout Step'.

An alternative would be to actually change the code to add an instruction note above or below the button, but that would not be the preferable way to provide instructions.

**Comment by lat9** [comment]
Thanks for the suggestions.  How about having a hover/title on the button that has that additional guidance?

**Comment by lindasdd** [comment]
The problem with that is one would need to hover to see it and hover/title doesn't work in mobile, so it only works well when using a question mark button that appears on click.

What I am doing on my site is just using the following custom language file declaration:
define('BUTTON_SAVE_CHANGES_ALT', 'Confirm Address and Continue Checkout');

This worked well for the other 2 instances that the button appears.

**Comment by lat9** [comment]
Good point, @lindasdd.  I'll note that you could also add (via that language-override file) some guidance regarding how the 'flow' goes:
```php
    // -----
    // This constant defines the instructions you want displayed at the very top of the "checkout_one" page, before the form entry.
    //
    define ('TEXT_CHECKOUT_ONE_TOP_INSTRUCTIONS', '');  //-Displayed within a set of <p>...</p> tags if not empty.
```

**Comment by lindasdd** [comment]
I did see (and appreciate) that section. 

I won't be using it just because I try to make the user experience seamless enough that instructions are only provided when there is no other way. If I have to provide instructions it means I haven't put enough effort into the design.
